packages/node-cli/src/CLI.js

```diff
@@ -43,7 +44,13 @@ module.exports = class CLI {
 			const commandMetadataList = this._commandsMetadataService.getCommandsMetadata();
 
 			const thirdArgument = process.argv[2];
-			if (thirdArgument && thirdArgument !== HELP_ALIAS && thirdArgument !== HELP_OPTION && thirdArgument !== VERSION_OPTION) {
+			if (
+				thirdArgument &&
+				thirdArgument !== HELP_ALIAS &&
+				thirdArgument !== HELP_OPTION &&
+				thirdArgument !== HELP_COMMAND &&
```

Comment:
Just to be more explicit and avoid confusion, I would be naming these constatnts HELP_OPTION, HELP_OPTITON_ALIAS, HELP_COMMAND